packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -1,10 +1,17 @@
 import fsExtra from "fs-extra";
 import { fileURLToPath } from "url";
+import type { ApplicationBuilderOptions } from "@angular-devkit/build-angular";
+import { spawn } from "child_process";
+import { resolve, normalize, relative } from "path";
+import { stringify as yamlStringify } from "yaml";
+import { OutputPathOptions } from "./interface.js";
 
 // fs-extra is CJS, readJson can't be imported using shorthand
-export const { readJson } = fsExtra;
+export const { writeFile, move, readJson } = fsExtra;
 
-export async function loadConfig(cwd: string) {
+export type { ApplicationBuilderOptions };
+
+export async function loadConfig(cwd: string): Promise<ApplicationBuilderOptions> {
   // dynamically load NextJS so this can be used in an NPX context
```

Comment:
not from this PR but this should say Angular instead

Reply:
removed this import to use build manifest